bincrafters_conventions/actions/update_c_attributes.py

```diff
@@ -0,0 +1,42 @@
+import re
+
+
+def update_c_topics(main, file):
+    """ Update topics tuple
+    """
+
+    conanfile = open(file).read()
+    match = re.search("""topics\\s*=\\s*(?P<topics>.*)""", conanfile)
+    if match:
+        original = match[0]
+        if match['topics'][0] == '(':
+            return False
+        topics = match['topics']
+
+        topics_iter = re.finditer('''(['"])(\\S*)\\1''', topics)
+        change = 'topics = ({})'.format(', '.join('{}{}{}'.format(t[1], t[2], t[1]) for t in topics_iter))
+
+        main.replace_in_file(file, original, change)
```

Comment:
If there is a successful replacement, please output an descriptive update message like in the other existing updates